### ISSUE TITLE
Rename / reconsider "good first issue" label

### ISSUE THREAD

**Issue by doismellburning** [SRCF/lightbluetent#23]
It's not entirely clear of the intent around the "good first issue" label but I think there's a mismatch between expectation created and the usage.

The description is "good for newcomers" - to me that suggests at least two criteria:

1. The work required to resolve the issue is simple
2. The work needed to *understand* the issue is simple

I think the label is being used for things that meet 1 but not 2

As a newcomer, I look at e.g. https://github.com/SRCF/lightbluetent/issues/20 and I have no idea what "Port basic helper functions for API calls" is supposed to even mean.

It might be the case that it's easily doable, but it's certainly not the case that it's easily understandable.

Assuming I've understood the goal correctly, then several of these issues would really benefit from explanatory context around what the actual issue is and what work is necessary.

Does that make sense?

**Comment by matiasilva** [comment]
I agree completely, will have a closer look at this later. The goal was to label the issues that are the simplest to takcle first.

**Comment by matiasilva** [comment]
Upon a closer read, I understand what this issue is getting at. When I added all these issues, I wasn't expecting many others to contribute (glad I'm wrong!) so I didn't go through and add context. I'll fix this by adding a simple issue template and elaborating on my previous issues. Thanks for bringing this up! If you have any other words, feel free to tack them on otherwise let's wait until I implement the above to close this.
